### PR TITLE
HTCONDOR-1033 Miron demo feedback.

### PR DESCRIPTION
**Pull request by Todd-L-Miller**
Lots of mostly minor changes from Miron's feedback.

# HTCondor Pull Request Checklist for internal reviewers

- [ ] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [x] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [ ] Verify that the branch destination of the PR matches the target version of the ticket
- [ ] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [ ] Check for documentation, if needed
- [ ] Check for version history, if needed
- [ ] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [ ] Check that commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: src/condor_tools/htcondor_cli/annex_create.py [hunk 1]
```diff
@@ -491,6 +496,34 @@ def annex_create(
     cpus,
     mem_mb,
 ):
+    if '@' in queue_at_machine:
+        (queue_name, target) = queue_at_machine.split('@', 2)
```
Comment (jasoncpatton) [inline src/condor_tools/htcondor_cli/annex_create.py:500]:
```suggestion
        (queue_name, target) = queue_at_machine.split('@', 1)
```

```python
>>> "queue@machine@other".split("@", 2)
['queue', 'machine', 'other']

>>> (queue, machine) = "queue@machine@other".split("@", 2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: too many values to unpack (expected 2)

>>> "queue@machine@other".split("@", 1)
['queue', 'machine@other']

>>> (queue, machine)
('queue', 'machine@other')
```

File: src/condor_tools/htcondor_cli/annex.py [hunk 2]
```diff
@@ -10,7 +10,8 @@
 from htcondor_cli.noun import Noun
 from htcondor_cli.verb import Verb
 
-# Most of the annex create code is stored in a separate file.
+# Most of the annex add/create code is stored in a separate file.
+from htcondor_cli.annex_create import annex_add
 from htcondor_cli.annex_create import annex_create
```
Comment (jasoncpatton) [inline src/condor_tools/htcondor_cli/annex.py:15]: Your choice, just a little cleaner
```suggestion
from htcondor_cli.annex_create import annex_add, annex_create
```

**Review by jasoncpatton — CHANGES_REQUESTED** [review]
LGTM, only change is that I assume you're passing the `maxsplit` arg to `queue_at_machine.split()` to shove any extra `@`s in the machine name and/or prevent `ValueError`s from getting thrown.

**Comment by jasoncpatton** [comment]
Also, should this target `master` or `V9_8_0-branch`?

**Comment by Todd-L-Miller** [comment]
OK, two things:
* I just accepted both of your suggestions, but forgot to put the JIRA ticket in either of them.
* This should be targeting `V9_8_0-branch`, not `master`.
I will create a new PR of the same branch but targeting the correct branch, and cherry-pick the suggestions over with the correct commit message.

**Comment by Todd-L-Miller** [comment]
... OK, that plan won't work.  I'll try another one shortly.

**Pull request closed without merging**